lib/src/models/chat_feedback_info.dart

```diff
@@ -0,0 +1,15 @@
+class ChatFeedbackInfo {
+  final String authorId;
+  final int rating;
+  final String? comment;
+
+  ChatFeedbackInfo({required this.authorId, required this.rating, this.comment});
```

Comment:
quibble: we usually follow Dart and Flutter's code style which puts constructors before properties.